periodic-table/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: PERIODIC TABLE 
+uuid: periodic-table
+prerequisites:
+  - 
+learning_objectives: 
+  - 
+references:
+    -
+abet_outcomes:
+  -  
+assessments: 
+  - 
+...
+# Periodic table
```

Comment:
The title will be published with the web interface if the "layout: node" metadata is included above.